src/routes/give-away/create.js

```diff
@@ -0,0 +1,51 @@
+import models from '../../models'
+
+export default {
+
+  method: 'post',
+  path: '/giveaways?',
+
+  requireAuthentication: true,
```

Comment:
I'm thinking that we restrict this to certain userIds only (basically creating admin accounts)

Reply:
That can certainly be done. Could also maybe have whitelisted IPs instead so we could use whatever account we want as long as it's from our computers? 🤔